main.py

```diff
@@ -154,6 +156,20 @@ def is_valid_tag(tag: Optional[str]) -> bool:
     return content
 
 
+def get_cover_art(filename: str) -> Optional[discord.File]:
+    tags = TinyTag.get(filename, image=True)
+    # get image data as bytes, make sure it doesn't go over 8MB
+    # This is a safe lower bound on the Discord upload limit of 8MiB
+    image_data = tags.get_image()
+    if image_data is None or len(image_data) > 8_000_000:
+        return None
+    image = Image.open(BytesIO(image_data))
```

Comment:
I tried, and it's possible to embed a non-image file inside of an mp3. Attempting to load this into Pillow will raise an `PIL.PIL.UnidentifiedImageError` error. We should catch this (ideally log it) and just return `None` if that happens.